Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -1062,7 +1066,14 @@ public override nint RowsInSection(UITableView tableview, nint section)
 			public override void Scrolled(UIScrollView scrollView)
 			{
 				if (_isDragging && scrollView.ContentOffset.Y < 0)
+				{
 					_uiTableViewController.UpdateShowHideRefresh(true);
+				}
+
+				if (_isDragging && scrollView.ContentOffset.Y < -10f && _uiTableViewController._usingLargeTitles && Device.Info.CurrentOrientation.IsPortrait())
```

Comment:
I created an issue around some observed behavior here with IsPortrait
https://github.com/xamarin/Xamarin.Forms/issues/2200
I noticed this behavior because if I swipe the notification window down then swipe it back up then this code no longer runs and the Pull Down To Refresh stops working...    I'm wondering here if it's more reliable to check if *UIApplication.SharedApplication.StatusBarOrientation*